Screenbox/Pages/PlayerPage.xaml

```diff
@@ -265,13 +282,46 @@
 
                 <TextBlock
                     x:Name="SubtitleText"
+                    d:Text="Lorem ipsum dolor sit amet, consectetur adipiscing elit"
                     Opacity="{ThemeResource TextFillColorSecondaryOpacity}"
                     OpticalMarginAlignment="TrimSideBearings"
                     Style="{StaticResource BodyTextBlockStyle}"
                     Text="{x:Bind ViewModel.Media.AltCaption, Mode=OneWay, FallbackValue={}}"
                     TextWrapping="NoWrap"
                     Visibility="{x:Bind ViewModel.Media.AltCaption, Mode=OneWay, Converter={StaticResource StringVisibilityConverter}, FallbackValue=Collapsed}" />
             </StackPanel>
+
+            <Button
+                x:Name="LivelyOptionsButton"
+                Grid.Column="4"
+                Width="44"
+                Height="44"
+                Margin="4,6,4,2"
+                Padding="0"
+                VerticalAlignment="Top"
+                AutomationProperties.Name="{x:Bind LivelyOptionsButton.(ToolTipService.ToolTip)}"
+                Style="{StaticResource SubtleButtonStyle}"
+                ToolTipService.ToolTip="{strings:Resources Key=VisualizerOptionsToolTip}">
```

Comment:
Not really an issue but we probably should consider removing the title bar right padding when in fullscreen as well, now that we have a button there.

![msedge_abSWyTwdi7](https://github.com/user-attachments/assets/a711b54d-0210-4a7c-96e3-db00d7e35fe4)

Reply:
> Now in narrow window sizes, the buttons take up too much real estate. Should we consider resizing the buttons to the standard button size for this layout?

Personally I think we should slightly reduce it in all layouts, or consider alternatives

> Not really an issue but we probably should consider removing the title bar right padding when in fullscreen as well, now that we have a button there.

AFAIK we can't since it's reserved

![imagem](https://github.com/user-attachments/assets/50905369-5c13-4a11-bf3c-4990067a1726)